ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -1263,20 +1263,22 @@ export default {
           },
         };
 
-        if (matches.effectId === '998')
+        const id = matches.effectId.toUpperCase();
```

Comment:
(Thanks for the fix!) I see what the issue is.  In order to understand output strings inside of a response, cactbot makes the assumption that anything that has the `// cactbot-builtin-response` string in it can be called with `matches = {}` and not have an error.  In practice, `matches.effectId` has to exist if the regex matched, but in tests and the config ui that's not always true.
```suggestion
        if (!matches.effectId)
          return;
        const id = matches.effectId.toUpperCase();
```